doc/source/_static/js/landing_page.js

```diff
@@ -70,11 +75,21 @@ function displayFamilies(familyCounts) {
     familyName.className = "family-name";
     familyName.textContent = family;
 
+    const iconImage = document.createElement("img");
+    iconImage.alt = `${family} icon`;
+    iconImage.className = "ansys-icon";
+    const iconName =
+      theme === "dark" && icon === "ansys-icon-light.svg"
```

Comment:
explanation of this below https://github.com/ansys/pyansys/pull/859/files#r1975515748